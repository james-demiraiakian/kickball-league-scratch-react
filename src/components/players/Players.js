import React from 'react';
import { Link } from 'react-router-dom';

export default function Players({ name, position, team_id, selected }) {
  return (
    <div>
      <h4>{name}</h4>

      {selected && (
        <>
          <h5>{position}</h5>
          <h5>Team Info</h5>
        </>
      )}
    </div>
  );
}
