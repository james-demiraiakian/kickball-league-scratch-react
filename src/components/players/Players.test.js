import { render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Players from './Players';

it('renders player detail', async () => {
  const { container } = render(
    <MemoryRouter>
      <Players name="Greorge" position="Over There" selected={true} />
    </MemoryRouter>
  );

  screen.getByText('Over There');

  expect(container).toMatchSnapshot();
});
