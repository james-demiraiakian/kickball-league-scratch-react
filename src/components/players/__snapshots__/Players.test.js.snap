// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders player detail 1`] = `
<div>
  <div>
    <h4>
      Greorge
    </h4>
    <h5>
      Over There
    </h5>
  </div>
</div>
`;
